protocols/directfund/directfund.go

```diff
@@ -237,14 +255,19 @@ func (o *Objective) Reject() (protocols.Objective, protocols.SideEffects) {
 
 // Update receives an ObjectiveEvent, applies all applicable event data to the DirectFundingObjectiveState,
```

Comment:
```suggestion
// Update receives an ObjectivePayload, applies all applicable data to the DirectFundingObjectiveState,
```
and similarly for other protocols